e3sm_diags/driver/utils/dataset_xr.py

```diff
@@ -482,7 +495,16 @@ def _open_climo_dataset(self, filepath: str) -> xr.Dataset:
         # No need to decode times because the climatology is already calculated.
         # Times only need to be decoded if climatology is being calculated
         # (time series datasets).
-        args = {"path": filepath, "decode_times": False, "add_bounds": ["X", "Y"]}
+        # NOTE: This GitHub issue explains why the "coords" and "compat" args
+        # are defined as they are below: https://github.com/xCDAT/xcdat/issues/641
+        args = {
+            "paths": filepath,
+            "decode_times": False,
+            "add_bounds": ["X", "Y"],
+            "coords": "minimal",
+            "compat": "override",
+            "chunks": "auto",
```

Comment:
```suggestion
```

Removing `chunks` because it is not needed since we call `.load()` to load the completel array into memory.

Reply:
Addressed in [`1bf91a6` (#798)](https://github.com/E3SM-Project/e3sm_diags/pull/798/commits/1bf91a6ca085255acc13f8affee88d369703d01a)